src/fired-events.js

```diff
@@ -0,0 +1,29 @@
+export function eventHandler (client, type) {
+    client.context.set('fired_events.'+type, true, true);
```

Comment:
Thinking about the design on this one -- initially I had thought we should have the javascript sdk responsible for populating the context when an event was fired, and asset manager handling the session storage. That may be a little hard for someone else to follow -- splitting the responsibilities between the 2 repos. Especially given that we need to rehydrate the context without firing the event again. So yeah, let's put this logic in the asset maanger.

Are you sure we don't cause an infinite loop here?